File: HomeCareApi/Homecare.Application/Constants/Auth/AuthMessages.cs
namespace Homecare.Application.Constants.Auth;

public static class AuthMessages
{
    public const string InvalidCredentials = "Invalid Credentials";
    public const string LoginSuccess = "Login successful";

    public const string InvalidRefreshToken = "Invalid refresh token";
    public const string TokenRefreshed = "Token refreshed successfully";

    public const string LogoutSuccess = "Logged out successfully";

    public const string EmailNotRegistered = "Email is not registered";
    public const string ResetLinkSent = "A Password reset link has been sent.";
    public const string ForgotPasswordEmailSubject = "Reset your Password";

    public const string PasswordMismatch = "Passwords do not match";
    public const string PasswordResetSuccess = "Password has been reset successfully";

    public const string InvalidResetLink = "Invalid reset link";
    public const string ResetLinkAlreadyUsed = "Reset link already used";
    public const string ResetLinkExpired = "Reset link expired";
    public const string ResetTokenValid = "Valid";
}

File: HomeCare-Admin/src/app/core/constants/auth-messages.ts
export const AUTH_MESSAGES = {
  LOGIN: {
    INVALID_CREDENTIALS: 'Invalid email or password',
    GENERIC_ERROR: 'Something went wrong. Please try again.',
  },

  FORGOT_PASSWORD: {
    INVALID_EMAIL: 'Please enter a valid email address.',
    GENERIC_ERROR: 'Something went wrong. Please try again.',
  },

  RESET_PASSWORD: {
    INVALID_LINK: 'Invalid link',
    INVALID_EXPIRED: 'Invalid or expired reset link',
    PASSWORD_MISMATCH: 'Passwords do not match.',
    INVALID_DETAILS: 'Please enter valid details.',
    GENERIC_ERROR: 'Something went wrong. Please try again.',
  },
} as const;
